app/models/ebean/NagiosExtension.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2015 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * Data model for extensions data in alerts.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+@Entity
+@Table(name = "nagios_extensions", schema = "portal")
+public class NagiosExtension extends AlertExtension {
+
+
+    @Column(name = "severity")
+    private String _severity;
+
+    @Column(name = "notify")
+    private String _notify;
+
+    @Column(name = "max_check_attempts")
+    private int _maxCheckAttempts;
+
+    @Column(name = "freshness_threshold")
+    private int _freshnessThreshold;
+
+    public String getSeverity() {
+        return _severity;
+    }
+
+    public void setSeverity(final String value) {
+        _severity = value;
+    }
+
+    public String getNotify() {
+        return _notify;
+    }
+
+    public void setNotify(final String value) {
+        _notify = value;
+    }
+
+    public int getMaxCheckAttempts() {
+        return _maxCheckAttempts;
+    }
+
+    public void setMaxCheckAttempts(final int value) {
+        _maxCheckAttempts = value;
+    }
+
+    public int getFreshnessThreshold() {
+        return _freshnessThreshold;
+    }
+
+    public void setFreshnessThreshold(final int value) {
+        _freshnessThreshold = value;
```

Comment:
I assume this is a time value, let's indicate its units in the member and column name.